dev/tools/localization/gen_l10n.dart

```diff
@@ -13,6 +13,8 @@ import 'gen_l10n_templates.dart';
 import 'gen_l10n_types.dart';
 import 'localizations_utils.dart';
 
+String defaultSyntheticPackagePath = path.join('.dart_tool', 'flutter_gen', 'gen_l10n');
```

Comment:
Could you leave a comment linking this path with the useSyntheticPackage setting in the flutter tool?